padd.sh

```diff
@@ -575,10 +575,11 @@ GetVersionInformation() {
     if [ "${padd_version_latest}" = "" ]; then
       padd_version_heatmap=${yellow_text}
     else
-      if [ "${padd_version}" != "${padd_version_latest}" ]; then
+      if [ "$(VersionConverter "${padd_version}")" -lt "$(VersionConverter "${padd_version_latest}")" ]; then
```

Comment:
Is that quoting as intended? Shell doesn't nest quotes.

Reply:
Shellcheck was complaining about double quoting to prevent word splitting. However, this might be some but in SC. If I change the variable name it does not complain anymore...

![Bildschirmfoto zu 2022-07-01 10-04-55](https://user-images.githubusercontent.com/26622301/176852825-0ce53dfa-4756-484c-b2f5-98b2b73de90a.png)

![Bildschirmfoto zu 2022-07-01 10-05-04](https://user-images.githubusercontent.com/26622301/176852831-819839ff-900b-4568-982b-664fe9dfabc1.png)

Reply:
@dschaper Do you have any idea why this single variable causes issues, but others not?
Might still be good to go....

Reply:
You'll need to translate the problem message for me. 😆

Reply:
In the lower screenshot (using `padd_version) it is: No issues detected. ;-)
Only in the upper, where it complains about double quoting when using `padd_version_latest`